### DISCUSSION TITLE
Pause/Resume job from appsetting.json

### DISCUSSION THREAD

**Discussion by SaeedGz98** [quartznet/quartznet#971] [General]
Hi, I have a curring job that execute every day at 5:00pm, for a reason i want to pause the executing of the specific job so i decided to add a flag as true/false in appsetting.json to handle this issue but i don't know how can I pause a job based on property value of appsetting

**Comment by lahma** [comment]
You should call `IScheduler.PauseJob` with the given job key.

**Reply by SaeedGz98** [reply]
but in runtime can't watch appsetting key value and apply pausejob when turns from true to false

**Reply by lahma** [reply]
That starts to sound quite complex. Maybe something around `IOptionsMonitor` and a reference to `IScheduler` to do the pause.
